bsdlib/doc/ug_bsdlib_porting_os.rst

```diff
@@ -1,12 +1,13 @@
 .. _bsdlib_ug_porting:
 
-User Guide: Porting the BSD library to your OS
-##############################################
+Porting the BSD library to your OS
+##################################
 
 The :ref:`bsdlib` from Nordic Semiconductor is an OS-agnostic C library.
-OS-specific parts are moved out and in order to use it with your OS or scheduler, you must first port it by creating a custom :file:`bsd_os.c` file which serves as an OS abstraction layer.
-The library exposes a :file:`bsd_os.h` header file which defines the functions that must be implemented in :file:`bsd_os.c`.
-The header file also exposes what functions are provided by the library that the OS integration module must interface.
+OS-specific parts are moved out of the library.
+To use it with your OS or scheduler, you must first port it by creating a custom :file:`bsd_os.c` file which serves as an OS abstraction layer.
```

Comment:
```suggestion
To use the library with your OS or scheduler, you must first port it by creating a custom :file:`bsd_os.c` file, which serves as an OS abstraction layer.
```

Reply:
Or that and no comma - in this case, both are okay. But not "which" without comma. ;)